systems/plants/@RigidBodyManipulator/forwardKinV.m

```diff
@@ -0,0 +1,159 @@
+function [x, J, dJ] = forwardKinV(obj, kinsol, body_or_frame_ind, points, rotation_type, base_or_frame_ind)
```

Comment:
This can probably be made a little bit more efficient based on my reimplementation in C++.